starkinfra/issuingtokendesign/__issuingtokendesign.py

```diff
@@ -0,0 +1,91 @@
+from starkcore.utils.resource import Resource
+from starkinfra.utils import rest
+from starkcore.utils.checks import check_datetime, check_date
+
+
+class IssuingTokenDesign(Resource):
+    """# IssuingTokenDesign object
+    The IssuingTokenDesign object displays the information of the Token Design created in your Workspace.
```

Comment:
`The IssuingTokenDesign object displays the information of the designs created in your Workspace. `